metadata.yaml

```diff
@@ -10,6 +10,7 @@ tags:
 subordinate: false
 series:
  - xenial
+ - bionic
```

Comment:
In my merge, I had left bionic out, and instead marked it as a canary for functional testing. Perhaps we should do that for a while before marking it supported?

Reply:
./agree I'll remove it